Codec/Archive/Zip.hs

```diff
@@ -188,6 +191,46 @@ data ZipState = ZipState
     -- ^ Pending actions
   }
 
+
+
```

Comment:
Every function should have description. The `Handle` argument should have it as well.